scripts/jenkins/qa/cloud-mkphyscloud-qa-scenario-6a.yaml

```diff
@@ -0,0 +1,178 @@
+
+      - string:
+          name: cct
+          default: features:base
+          description: Optional, specify cct tests to run
+
+      - string:
+          name: ssl_type
+          default: no-ssl
+          description: "Mandatory, set the SSL configuration for the cloud, available options: no-ssl, ssl-insecure, ssl"
+
+      - string:
+          name: cloud_version
+          default: "7"
+          description: Mandatory, version of the cloud to be installed as integer
+
+      - string:
+          name: scenario_url
+          default: https://raw.githubusercontent.com/SUSE-Cloud/automation/master/scripts/scenarios/cloud$cloud_version/qa/$ssl_type/qa-scenario-6a.yaml
+          description: Location of scenario yaml file
+
+      # Parameters for qa_crowbarsetup.sh
+      - string:
+          name: cloudsource
+          default: develcloud$cloud_version
+      - string:
+          name: TESTHEAD
+          default: "1"
+          description: if non-empty, test latest version from Devel:Cloud Stagin
+      - string:
+          name: hacloud
+          default: "1"
+          description: By default we do not want HA configured and installed
+      - string:
+          name: clusterconfig
+          default: services=2
+          description: HA configuration for clusters. Make sense only if hacloud=1
+      - string:
+          name: nodenumber
+          default: "7"
+          description: Number of nodes to use. Depends on hw_number
+      - string:
+          name: want_ipmi
+          default: "true"
+      - string:
+          name: runner_url
+          default: https://raw.githubusercontent.com/SUSE-Cloud/automation/master/scripts/qa_crowbarsetup.sh
+          description: The runner must be qa_crowbarsetup.sh file
+      - string:
+          name: commands
+          default: prepareinstallcrowbar installcrowbar allocate waitcloud setup_aliases batch
+          description: All the steps that needs to be completed to have cloud installed
+
+    builders:
+      - shell: |
+          #!/bin/bash
+          admin=crowbar$hw_number
+          cloud=qa$hw_number
+
+          if [ ! -z "$UPDATEREPOS" ] ; then
+            # testing update only makes sense with GMx and without TESTHEAD
+            unset TESTHEAD
+            export UPDATEREPOS=${UPDATEREPOS//$'\n'/+}
+          fi
+
+          export artifacts_dir=$WORKSPACE/.artifacts
+          rm -rf $artifacts_dir
+          mkdir -p $artifacts_dir
+          touch $artifacts_dir/.ignore
+          freshadminvm_cloud6 $admin qcow2
+          sleep 100 # time for the admin VM to boot
+
+          # wipe out shared NFS that should be used by this deployment
+          ssh root@localhost "wipe_nfs_shares qa$hw_number"
+
+          # rest of code runs on admin node:
+          env | grep -e networking -e libvirt -e cloud > mkcloud.config
+          scp mkcloud.config root@$admin:
+          ret=0
+
+          ssh root@$admin "
```

Comment:
I am a bit lost with this amount of commands that we send via ssh. Looks like that we need to scape some quotes, and not sure where it ends.

Can we use this << EOF stream? http://stackoverflow.com/questions/4412238/whats-the-cleanest-way-to-ssh-and-run-multiple-commands-in-bash